nio/client/async_client.py

```diff
@@ -917,6 +918,29 @@ def add_response_callback(
 
         return await self._send(KeysQueryResponse, method, path, data)
 
+    @logged_in
+    async def devices(self):
+        # type: () -> Union[DevicesResponse, DevicesError]
+        method, path = Api.devices(self.access_token)
+
+        return await self._send(DevicesResponse, method, path)
+
+    @logged_in
+    async def delete_devices(self, devices, auth=None):
```

Comment:
Docs missing. Since this uses the interactive auth thing, which isn't completely trivial, an example in the docs would be nice as well.